source/codegen/metadata/nidmm/enums.py

```diff
@@ -1,83 +1,88 @@
-﻿# -*- coding: utf-8 -*-
-# This file is generated from NI-DMM API metadata version 19.6.0d7
+# -*- coding: utf-8 -*-
+# This file is generated from NI-DMM API metadata version 22.8.0d9999
 enums = {
-    'AcquisitionStatus': {
+    'ADCCalibration': {
+        'codegen_method': 'public',
         'values': [
             {
-                'name': 'NIDMM_VAL_RUNNING_ANTICOLLISION',
+                'name': 'NIDMM_VAL_ADC_CALIBRATION_OFF',
                 'value': 0
             },
             {
-                'name': 'NIDMM_VAL_FINISHED_WITH_BACKLOG',
-                'value': 1
-            },
-            {
-                'name': 'NIDMM_VAL_FINISHED_WITH_NO_BACKLOG',
-                'value': 2
-            },
-            {
-                'name': 'NIDMM_VAL_PAUSED',
-                'value': 3
+                'name': 'NIDMM_VAL_ADC_CALIBRATION_AUTO',
+                'value': -1
             },
             {
-                'name': 'NIDMM_VAL_NO_ACQUISITION_IN_PROGRESS',
-                'value': 4
+                'name': 'NIDMM_VAL_ADC_CALIBRATION_ON',
+                'value': 1
             }
         ]
     },
-    'AdcCalibration': {
+    'AcquisitionStatus': {
+        'codegen_method': 'public',
         'values': [
             {
-                'name': 'NIDMM_VAL_ADC_CALIBRATION_OFF',
+                'name': 'NIDMM_VAL_ACQUISITION_STATUS_RUNNING',
                 'value': 0
             },
             {
-                'name': 'NIDMM_VAL_ADC_CALIBRATION_AUTO',
-                'value': -1
+                'name': 'NIDMM_VAL_ACQUISITION_STATUS_FINISHED_WITH_BACKLOG',
+                'value': 1
             },
             {
-                'name': 'NIDMM_VAL_ADC_CALIBRATION_ON',
-                'value': 1
+                'name': 'NIDMM_VAL_ACQUISITION_STATUS_FINISHED_WITH_NO_BACKLOG',
+                'value': 2
+            },
+            {
+                'name': 'NIDMM_VAL_ACQUISITION_STATUS_PAUSED',
+                'value': 3
+            },
+            {
+                'name': 'NIDMM_VAL_ACQUISITION_STATUS_NO_ACQUISITION_IN_PROGRESS',
+                'value': 4
             }
         ]
     },
     'ApertureTime': {
+        'codegen_method': 'public',
+        'generate-mappings': True,
```

Comment:
After [this PR](https://dev.azure.com/ni/DevCentral/_git/ni-central/pullrequest/361109) goes in and I regen from hapigen, the values should go back to integers and `'generate-mappings': False,`